formbricks/oss.gg
# feat: Post Discord channel message on points earned

## What does this PR do?

1. Posts discord channel message whenever someone earns new points



Fixes #75 

https://github.com/formbricks/oss.gg/assets/66999381/b9467d00-0d39-43e5-9689-bd97b3e613e3




## How should this be tested?



- Create a discord bot and install it in the server.
- Get the `bot token` and `channel ID` and add the env variables
- Award points and the discord message should be posted

## Checklist



### Required

- [x] Read [How we Code at oss.gg](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)

### Appreciated
